addons/website_sale/controllers/main.py

```diff
@@ -1249,6 +1249,10 @@ def shop_payment_transaction(self, order_id, access_token, **kwargs):
         except AccessError:
             raise ValidationError("The access token is invalid.")
 
+        order = request.env['sale.order'].browse(order_id)
+        if order.state == 'cancel':
+            raise ValidationError('The order has been canceled.')
```

Comment:
The error must be translatable.
```suggestion
            raise ValidationError(_("The order has been canceled."))
```
Note that you'll need to [update the module's translations](https://www.odoo.com/documentation/master/developer/howtos/translations.html#exporting-translatable-term) in this PR.

Reply:
Thanks! It's done.
I took the liberty of adding another commit to make the other ValidationError in this method translatable too